src/appleseed/renderer/kernel/lighting/lighttree.cpp

```diff
@@ -208,7 +208,17 @@ float LightTree::recursive_node_update(
             // Retrieve the emitting triangle importance.
             const EDF* edf = triangle.m_material->get_uncached_edf();
             assert(edf != nullptr);
-            importance = edf->get_uncached_max_contribution() * edf->get_uncached_importance_multiplier();
+
+            const float max_contribution = edf->get_uncached_max_contribution();
+
+            // max_contribution is reported as std::numeric_limits<float>::max() when
+            // we can't compute the max_contribution easily (ex: textured lights)
+            // In such cases, we can use a default importance value of 1.0 to avoid
+            // infinite importance values in the light tree nodes.
+            if (max_contribution == std::numeric_limits<float>::max()) 
```

Comment:
Makes a lot of sense.

You'll need to include `<limits>`, and you can remove the `std` qualifier since there's a using clause at the top of the file.

Reply:
Ooops, missed that. Will fix it.